# Common type for switch commands

---
bool confirmation
